libsplinter/src/admin/rest_api/mod.rs

```diff
@@ -22,10 +22,11 @@ use std::time;
 
 use crate::actix_web::{web, HttpResponse};
 use crate::futures::{Future, IntoFuture};
+use crate::protocol;
```

Comment:
Have stories been added to add the guard to the other routes as well?

Reply:
This is not the complete solution, with respect to guarding the routes. It is merely the mechanism to do so.  The task in question will be completed with a follow-on PR